client/my-sites/stats/activity-log/index.jsx

```diff
@@ -100,6 +100,13 @@ class ActivityLog extends Component {
 		window.scrollTo( 0, 0 );
 	}
 
+	handlePeriodChange = ( { date, direction } ) => {
+		this.props.recordTracksEvent( 'calypso_activitylog_monthpicker_change', {
+			date,
```

Comment:
Let's make sure to use a string representation of the date, and not a `Moment` object - nested properties are not supported by tracks.